etc/grafana/dashboards/overview.json

```diff
@@ -7276,7 +7501,11 @@
   "templating": {
     "list": [
       {
-        "current": {},
+        "current": {
+          "selected": false,
+          "text": "localhost:9005",
+          "value": "localhost:9005"
+        },
```

Comment:
not sure it should be here, are these residues of the public nodes?

Reply:
I just copied the entire dashboard json. will remove.